homeassistant/components/sensor/htu21d.py

```diff
@@ -39,12 +39,11 @@
 })
 
 
-# pylint: disable=import-error
 @asyncio.coroutine
 def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     """Set up the HTU21D sensor."""
-    import smbus
-    from i2csense.htu21d import HTU21D
+    import smbus                        # pylint: disable=import-error
```

Comment:
Separating with two spaces from the statement looks like enough and aligning with the next comment seems unnecessary.

Reply:
Addressed.